packages/core/admin/admin/src/content-manager/components/BlocksEditor/BlocksInput/index.js

```diff
@@ -74,6 +74,26 @@ const BlocksInput = ({ disabled, placeholder }) => {
     }
   };
 
+  /**
+   * Modifier keyboard shortcuts
+   */
+  const handleKeyboardShortcuts = (event) => {
+    const isCtrlOrCmd = event.metaKey || event.ctrlKey;
```

Comment:
I think this would make the shortcut on windows "windows key + event.key", instead of "ctrl + event.key". Because on windows the metaKey is the windows key.

If it's confirmed then I would go back to your 1st implementation since navigator.platform is deprecated

Reply:
I would probably do the inverse of this, but pressing the windows key summons things on the OS level so I wouldn't be worried about it triggering. The issue with using a regex to detect a user agent is that you have to keep that up to date and you're actually only assuming there are two OS' which in fact for a desktop there are _many_ iirc linux comes in many flavours.

Reply:
Agree about regex, I will confirm with product having both keys working.

Reply:
I just confirmed with Yannis, its fine to have both combinations working.